source-code/index.md

```diff
@@ -245,10 +245,6 @@ facilitated by the code in the `raml-module-builder` repository.
 - [mod-rmb-template](https://github.com/folio-org/mod-rmb-template)
   -- A Maven archetype to commence a new RMB-based module.
 
-- [mod-pg-embed](https://github.com/folio-org/mod-pg-embed)
```

Comment:
@dcrossleyau : Can we (in a GitHub sense) _archive_ `folio-org/mod-pg-embed`?  I think we'll stop getting security updates for it then.

Reply:
Too late. It was actually "deleted" around Feb 5 -- https://issues.folio.org/browse/FOLIO-1649

You might be looking at an old security report. I do not see it listed since that week.

We are working out a process for deprecating repos: https://issues.folio.org/browse/FOLIO-1710